scripts/uosc/elements/Element.lua

```diff
@@ -154,4 +161,28 @@ function Element:flash()
 	end
 end
 
+-- Register disposer to be called when element is destroyed.
+---@param disposer fun()
+function Element:register_disposer(disposer)
+	if not itable_index_of(self._disposers, disposer) then
```

Comment:
The disposer is always a new function, so `itable_index_of()` will never find anything.

Reply:
It is a new function in current implementations, but it doesn't have to be in future ones. And it's not a good idea to call disposers twice.